### ISSUE TITLE
Joycons dont work on catalina?

### ISSUE THREAD

**Issue by volaix** [SymenYang/Magic-Joy-Con#2]
Hey, I'm assuming this doesn't work on Catalina? (Have given access in accessibility).

Anyone else got it working?

**Comment by aikinai** [comment]
Also not working in Big Sur.

**Comment by gingerbeardman** [comment]
Also not working in Monterey.

Interesting to see it does light up the correct attached controller in the toolbar icon.
